daemon/indexing/apollo/index.js

```diff
@@ -195,7 +196,7 @@ const typeDefs = gql`
 
   # The "Query" type is the root of all GraphQL queries.
   type Query {
-    listings(searchQuery: String, filters: [ListingFilter!]): ListingPage, #(searchQuery: String, page: Page, order: ListingOrder, filters: [ListingFilter!]): ListingPage,
+    listings(searchQuery: String, category: String, filters: [ListingFilter!]): ListingPage,
```

Comment:
Why is category a string vs added as a ListingFilter ?
Also, we'll need to add filter by subCategory.

Reply:
Yeah good point. I have changed it to be added as a `ListingFilter` like suggested. Subcategory is already added as a `ListingFilter`.

Reply:
Still there ?